lib/Container/CoreExtension.php

```diff
@@ -74,6 +74,19 @@ public function load(Container $container)
         $this->registerApplicationServices($container);
     }
 
+    private function getBaseCwd($path = null) {
+        if (is_null($path)) {
+            $path = getcwd();
+        }
+
+        // Return base CWD where .git directory is present
+        if (!is_dir($path . '/' . '.git') and $path !== '/') {
```

Comment:
Could use `sprintf('%s/.git')` and use `&&` instead of `and`

Reply:
Done !